rero_ils/modules/notifications/mappings/v6/notifications/notification-v0.0.1.json

```diff
@@ -15,14 +15,20 @@
         "pid": {
           "type": "keyword"
         },
-        "notification_date": {
+        "loan_pid": {
```

Comment:
do not for forget to update mapping after jsonschema notifications

Reply:
done